.github/ISSUE_TEMPLATE/problem_report.md

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
I think that we should have two issue templates, one to report an issue with the stale action (ex: `stale_issue_report.md`) and one very generic to handle any other kind of issue (`other_issue_report.md`).
You could keep the current template to work as the stale issue report.
And the comments below aim to improve it.